### PR TITLE
Always use locally installed jq binary

### PR DESCRIPTION
**Pull request by mackermans**
@davesnx @wellsjo 
This PR should solve https://github.com/sanack/node-jq/issues/47

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
   },
   "scripts": {
     "install-binary": "node scripts/install-binary.js",
-    "pretest": "npm run install-binary",
     "test": "mocha --compilers js:babel-register",
```
Comment (davesnx) [inline package.json]: I think you need to on pretest install the binary

Comment (mackermans) [inline package.json]: We could also add a check in `pretest` to see if the binary is installed already, and if not, run this script. What do you think?

Comment (davesnx) [inline package.json]: I will update the install-jq binary

File: scripts/install-binary.js [hunk 2]
```diff
@@ -3,22 +3,40 @@
...
+const tempfile = require('tempfile')
+const fs = require('fs')
 
 const JQ_INFO = {
   name: 'jq',
   url: 'https://github.com/stedolan/jq/releases/download/',
   version: 'jq-1.5'
 }
 
+const OUPUT_DIR = path.join(__dirname, '..', 'bin')
```
Comment (mackermans) [inline scripts/install-binary.js]: Typo: `OUPUT_DIR` => `OUTPUT_DIR`

Comment (davesnx) [inline scripts/install-binary.js]: xD

**Pull request merged**